sdk/translation/ai-translation-text/review/cognitiveservices-translator.api.md

```diff
@@ -0,0 +1,716 @@
+## API Report File for "@azure-rest/ai-translation-text"
```

Comment:
Looks like a duplicate file, could you please delete it?

Reply:
Deleted